source/server/listener_manager_impl.cc

```diff
@@ -124,42 +124,45 @@ class ListenerSocketOption : public Network::Socket::Option, Logger::Loggable<Lo
     UNREFERENCED_PARAMETER(state);
 
     const auto* ip = socket.localAddress()->ip();
-    if (!ip) {
+    if (ip == nullptr) {
       // Do not fail when transparent option is not requested.
-      return transparent_ ? false : true;
+      return transparent_ == false ? true : false; // Easier to read than just '!transparent_'
```

Comment:
I think it would be easier to read is just "return !transparent_;"

Reply:
Will do.